README.md

```diff
@@ -42,6 +42,8 @@ The traces generation parameters could be simply provided via command line optio
 - `--traceTypesCount=5` - Number of traces types for auto-generation. Default: 0
 - `--totalTraceCount=16` - Total number of traces for generation. If this parameter greater
  than 0, `duration` will be ignored. Default: 0
+- `--stat output.json` - To dump statistics about generated traces to the given file in JSON format.
```

Comment:
The tool could produce multiple output files (traces to file, statistics to file, maybe something else later). I guess it will be better to name files according to their content, eg stats.json, or stats_output.json, or something else like this.